veggieseasons/analysis_options.yaml

```diff
@@ -0,0 +1,9 @@
+include: package:pedantic/analysis_options.yaml
+
+linter:
+  rules:
+    - avoid_types_on_closure_parameters
+    - camel_case_types
+    - constant_identifier_names
+    - non_constant_identifier_names
+    - unnecessary_new
```

Comment:
I think we could also consider adding the additional Flutter lints marked as such on http://dart-lang.github.io/linter/lints/: 

- cancel_subscriptions
- close_sinks
- control_flow_in_finally
- empty_statements
- hash_and_equals
- test_types_in_equals
- throw_in_finally
- unnecessary_statements
- await_only_futures
- implementation_imports
- package_api_docs
- package_prefixed_library_names
- unnecessary_brace_in_string_interps
- unnecessary_getters_setters
- package_names

Reply:
That's a longer list than I was hoping to add, but I don't see anything in there I disagree with. @domesticmouse, thoughts?

Reply:
Based on a conversation I just had with @bratay, I'd like to nominate avoid_void_async as well:

https://dart-lang.github.io/linter/lints/avoid_void_async.html

Reply:
I'm all for adding more lints, as it makes our code more consistent and predictable. I've spent enough time in go-land to enjoy the predictability that a strict compiler and formatter can bring to the community's code.